salt/metalk8s/kubernetes/apiserver/installed.sls

```diff
@@ -80,15 +80,13 @@ Set up default basic auth htpasswd:
 {%- set host = grains['metalk8s']['control_plane_ip'] %}
 
 Create kube-apiserver Pod manifest:
-  file.managed:
+  metalk8s.static_pod_managed:
     - name: /etc/kubernetes/manifests/kube-apiserver.yaml
     - source: salt://metalk8s/kubernetes/files/control-plane-manifest.yaml
-    - template: jinja
-    - user: root
-    - group: root
-    - mode: 644
-    - makedirs: True
-    - dir_mode: 750
+    - config_files:
+        - {{ htpasswd_path }}
+        - /etc/keepalived/check-apiserver.sh
+        - /etc/keepalived/keepalived.conf.sh
```

Comment:
Also API-server certificates (CA, server cert, `kubelet` client cert,...)

Reply:
OK, I was wondering about certs

Reply:
What about public/private keys? I added `sa.pub` for `k-a` but was not sure whether to store the private key digest (security?)...

Reply:
Let's assume the digest function is strong enough not to be reversible... It's the MD5 of a concatenation of SHA256s of the inputs, should be fine ;)